src/Uno.UI.Tests/Windows_UI_XAML_Controls/ContentDialogTests/Given_ContentDialog.cs

```diff
@@ -16,23 +16,22 @@ public class Given_ContentDialog
 		public void Init()
 		{
 			UnitTestsApp.App.EnsureApplication();
-
-			// The real Style contains a template, which causes ContentControl.IsContentPresenterBypassEnabled to return false. This is a workaround
-			// which shouldn't be necessary once Styles handling is overhauled.
-			Style.RegisterDefaultStyleForType(typeof(ContentDialog), new Style());
 		}
 
 		[TestMethod]
 		public void When_Has_DataContext()
 		{
+			var app = UnitTestsApp.App.EnsureApplication();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'app'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826539)